README.md

```diff
@@ -29,7 +29,7 @@ Add a workflow file to your repository to create custom automated processes.
   and `success` steps. If not provided defaults to false.
 - DEBUG: [Optional] Boolean value expected. Outputs debug information for
   semantic-release plugins. If not provided, defaults to false.
-- SCRIPT_PATH: [Optional] String value expected. Executes script with
+- SCRIPTS_PATH: [Optional] String value expected. Executes script with
```

Comment:
I think that the previous `SCRIPT_PATH` was actually correct, as you can only provide one script here. My previous comment was in regards to `"script" directory`, which in our case is always `"scripts"`, but we also just aligned to remove this limitation. Sorry for the confusion.